File: Java/1494.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws Exception {
		Scanner sc = new Scanner(System.in);
		
		int n, k, s, e, u, sum = 0, i;
		n = sc.nextInt();
		k = sc.nextInt();
		int arr[] = new int[n + 1];
		
		for(i = 0; i < k; i++) {
			s = sc.nextInt();
			e = sc.nextInt();
			u = sc.nextInt();
			
			arr[s - 1] += u;
			arr[e] -= u;
		}
		
		// Ãâ·Â
		for(i = 0; i < n; i++) {
			System.out.printf("%d ", arr[i]);
		}
		System.out.println();
		for(i = 0; i < n; i++) {
			sum += arr[i];
			System.out.printf("%d ", sum);
		}
	}
}

File: C/1494.c
#include <stdio.h>

int main(){
    int n, k, s, e, u, sum = 0, i;
    scanf("%d %d", &n, &k);
    int arr[n];
    for(i = 0; i < n; i++) {
        arr[i] = 0;
    }
    
    for(i = 0; i < k; i++) {
        scanf("%d %d %d", &s, &e, &u);
        arr[s - 1] += u;
        arr[e] -= u;
    }
    
    // Ãâ·Â
    for(i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
    printf("\n");
    for(i = 0; i < n; i++) {
        sum += arr[i];
        printf("%d ", sum);
    }
    
	return 0;
}